adam-core/src/main/scala/org/bdgenomics/adam/rdd/variation/VariantContextRDD.scala

```diff
@@ -111,9 +112,18 @@ case class VariantContextRDD(rdd: RDD[VariantContext],
    *
    * @param filePath The filepath to save to.
    * @param sortOnSave Whether to sort before saving. Default is false (no sort).
+   * @param sortLexicographically If true and sortOnSave is true, the variants
+   *   will be sorted by lexicographically ordered contig name, instead of by
+   *   sequence index order. If sortOnSave is false, this parameter is ignored.
+   *   Default is false (use contig index order).
+   * @param asSingleFile If true, saves the output as a single file by merging
+   *   the sharded output after completing the write to HDFS. If false, the
+   *   output of this call will be written as shards, where each shard has a
+   *   valid VCF header. Default is false.
    */
   def saveAsVcf(filePath: String,
```

Comment:
I kinda agree. I think it would probably make sense to break this out as is done with `AlignmentRecord`s and have `sortByXYZ` functions.